data/net/minecraft/world/level/levelgen/synth/NormalNoise.mapping

```diff
@@ -1,13 +1,20 @@
 CLASS net/minecraft/world/level/levelgen/synth/NormalNoise
+	COMMENT This samples the sum of two individual samplers of perlin noise octaves.
+	COMMENT The input coordinates are scaled by {@link #INPUT_FACTOR}, and the result is scaled by {@link #valueFactor}.
```

Comment:
```suggestion
	COMMENT The input coordinates are scaled by {@link net.minecraft.world.level.levelgen.synth.NormalNoise#INPUT_FACTOR}, and the result is scaled by {@link net.minecraft.world.level.levelgen.synth.NormalNoise#valueFactor}.
```

Reply:
See prior, they're owned by the current class so FQN is unnecessary for the link tag to work.